server/services/conversation.ts

```diff
@@ -114,8 +115,21 @@ export default class ConversationService extends EventEmitter {
         let participantIds: DBObjectId[] = game.galaxy.players.map(p => p._id);
 
         let newConvo = getNewConversation(game, null, name, participantIds);
-
+        
+        let newMessage: ConversationMessage = {
+            _id: mongoose.Types.ObjectId(),
```

Comment:
Ideally we should not provide an id and should let mongo do this for us.

I assume you did this because typescript wanted you to.

Update the type to be `_id?: DbObjectId` to make it optional and remove the new id line.